types/component/AnonymousChat/BaseChatItem.types.ts

```diff
@@ -22,9 +22,7 @@ export interface BaseChatItemProps {
 
 export interface ChatItemMyTextProps extends BaseChatItemProps {
   status?: ChatStatus;
-  AnonymousImage: React.ReactElement;
   chatType?: string;
-  avatar?: string;
 }
 
 export interface ChatItemTargetText extends BaseChatItemProps {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing `AnonymousImage` and `avatar` from `ChatItemMyTextProps` is a good move if these properties are no longer used. However, please make sure that there are no remaining usages of these properties in your codebase to avoid potential runtime errors.

```diff
-  AnonymousImage: React.ReactElement;
-  avatar?: string;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->